k8s-cluster/base/admin-ui/configmaps.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: v1
 data:
-  firefox_ext_local_path: /tmp/zenhub-firefox
+  firefox_ext_local_path: /data/uploads/extensions/firefox
   firefox_ext_s3_bucket_dir: ff-extension
   next_public_unsigned_dir: extension
   nextauth_url: https://$(admin_ui_subdomain).$(domain_tld)
+  node_tls_reject_unauthorized: "0"
```

Comment:
This should NOT be used in production. @dylanmartyn @mmurilo This was for local development testing only.

Reply:
```suggestion
  node_tls_reject_unauthorized: "1"
```